openage/convert/changelog.py

```diff
@@ -35,6 +35,7 @@
     {"graphics"},
     {"interface"},
     {"interface"},
+    {"gamedata"},
```

Comment:
We need to remove the `gamedata` folder when gamedata is regenerated. That's the easiest and correct way i think. Btw the correct changelog name is "metadata", which doesn't quite fit.
So if we do
``` python
import shutil
shutil.rmtree('/path/to/your/dir/')
```
in `driver.py` right after
``` python
    if args.flag("no_metadata"):
        return
```

it will always purge the folder first.